tests/tests_host_to_unmanaged_host.yml

```diff
@@ -0,0 +1,102 @@
+---
+# yamllint disable rule:line-length
+- hosts: all
+
+  vars:
+    num_hosts: 2
+    main_hostname: "mainhost.local"
+    unmanaged_host: "{{ 'host0' + (num_hosts+1)|string + '.local' }}"
+    unmanaged_hostname: '192.168.122.103'
```

Comment:
Does this need to be an unresolvable ip address?  If so, does it have to be a valid ip address?   e.g. you could use a bogus value like `999.999.999.999` which is guaranteed not to resolve (but will cause problems with ip address parsers and might cause problems with resolvers).